packages/eslint-plugin/docs/rules/no-extraneous-class.md

```diff
@@ -18,18 +25,16 @@ Examples of code for this rule:
 ### ❌ Incorrect
```

Comment:
@goktugyil https://github.com/typescript-eslint/typescript-eslint/issues/5040#issuecomment-1134013764:

> Can you put correct and incorrect examples for this? 

How do these look to you?

Reply:
They're excellent